common/src/main/java/com/wynntils/features/inventory/ContainerSearchFeature.java

```diff
@@ -215,22 +215,24 @@ private void addSearchWidget(AbstractContainerScreen<?> screen, int renderX, int
     private void matchItems(String searchStr, AbstractContainerScreen<?> screen) {
         String search = searchStr.toLowerCase(Locale.ROOT);
 
-        NonNullList<ItemStack> playerItems = McUtils.inventory().items;
-        for (ItemStack itemStack : screen.getMenu().getItems()) {
-            Optional<WynnItem> wynnItemOpt = Models.Item.getWynnItem(itemStack);
-            if (wynnItemOpt.isEmpty()) continue;
-            if (playerItems.contains(itemStack)) continue;
-
-            String name = StyledText.fromComponent(itemStack.getHoverName())
-                    .getStringWithoutFormatting()
-                    .toLowerCase(Locale.ROOT);
-
-            boolean filtered = !search.isEmpty() && name.contains(search) && itemStack.getItem() != Items.AIR;
-            wynnItemOpt.get().getCache().store(WynnItemCache.SEARCHED_KEY, filtered);
-            if (filtered) {
-                autoSearching = false;
-            }
-        }
+        System.out.println(screen.getMenu().slots);
+        screen.getMenu().slots.stream()
+                .filter(slot -> slot.x <= 6 * 18 + 8 && slot.y <= 6 * 18)
```

Comment:
have you verified that this is correct for misc bucket, block bank etc?

Reply:
I think it would be better if we do something like the ItemAnnotator for container

Reply:
> have you verified that this is correct for misc bucket, block bank etc?

I forgot that it not only searches in your bank -.-
Im not in a guild so I can't verify that it works in the guild bank/member list even if I implement a solution

Reply:
But you can do the rest of the searchable containers, and then someone with guild access can test the guild stuff

Reply:
Also, I agree that we need a more comprehensive container model. It was an interesting idea to use the ItemAnnotator approach, I'll need to ponder that one for a while.

Nevertheless, that is going to require some effort to transform the current state of the code to a better model. But we will help that future change by not having magic numbers or special dances like this spread all over the place. So at the very least, create a function that e.g. streams the valid slot numbers for your bank account in ContainerModel.

Reply:
I can add a function parameter to the `SearchableContainer` enum